### PR TITLE
feat: [DSSC#866a2qajx] Integration des modifications

### PR DESCRIPTION
**Pull request by aertyuiop12345**
Closes #

#### What has been done to verify that this works as intended?
Aucun test unitaires n'a été fait sur les nouvelles insertions de code
#### Why is this the best possible solution? Were any other approaches considered?
non
#### How does this change affect users? Describe intentional changes to behavior and behavior that could have accidentally been affected by code changes. In other words, what are the regression risks?
Aucun risque de regression.
#### Do we need any specific form for testing your changes? If so, please attach one.
Aucun
#### Does this change require updates to documentation? If so, please file an issue [here]( https://github.com/getodk/docs/issues/new) and include the link below.
Non
#### Before submitting this PR, please make sure you have:
- [ ] run `./gradlew checkAll` and confirmed all checks still pass OR confirm CircleCI build passes and run `./gradlew connectedDebugAndroidTest` locally.
- [x] verified that any code or assets from external sources are properly credited in comments and/or in the [about file](https://github.com/getodk/collect/blob/master/collect_app/src/main/assets/open_source_licenses.html).
- [x] verified that any new UI elements use theme colors. [UI Components Style guidelines](https://github.com/getodk/collect/blob/master/docs/CODE-GUIDELINES.md#ui-components-style-guidelines)

### CODE REVIEW

File: collect_app/src/main/java/org/odk/collect/android/activities/LandingPageActivity.java [hunk 1]
```diff
@@ -124,6 +135,26 @@ private void initToolbar() {
...
...serez redirigée vers Google Play Store pour l’installer.\n\nCette application utilise la méthode Knaus-Ogino, une méthode de contraception naturelle, pour déterminer les jours fertiles, mais elle présente un taux d'échec supérieur à 14%. L'abstinence sexuelle est la seule méthode de contraception efficace à 100% pour éviter complètement une grossesse.\n\nBien qu'elle donne une indication sur les jours non fertiles, il n’est pas recommandé d’utiliser cette application pour éviter une grossesse. Les utilisatrices doivent s'adresser aux formations sanitaires habilitées si elles ont besoin de conseils sur l'utilisation de cette application avec des méthodes contraceptives modernes.\n\nLes informations fournies par cette application ne doivent pas être considérées comme des conseils médicaux.\n\nL’objectif principal de cette application est d'aider les femmes à suivre leur cycle et santé menstruel.\n\nAfin d'éviter tout risque de préjudice ou d'erreur de diagnostic, il est fortement recommandé aux utilisatrices de ne pas tenter de s'auto-diagnostiquer ou de s'auto-traiter. En cas de symptômes ou d'irrégularités identifiés, elles doivent consulter des formations sanitaires habilitées.");
```
Comment (Baom92) [inline collect_app/src/main/java/org/odk/collect/android/activities/LandingPageActivity.java]: Mettre ces textes là dans le fichiers strings.xml

File: collect_app/src/main/java/org/odk/collect/android/activities/LandingPageActivity.java [hunk 2]
```diff
@@ -124,6 +135,26 @@ private void initToolbar() {
...
+        builder.setIcon(R.drawable.notes);
+        builder.setPositiveButton("J’ai lu et bien", new DialogInterface.OnClickListener() {
```
Comment (Baom92) [inline collect_app/src/main/java/org/odk/collect/android/activities/LandingPageActivity.java]: Ce texte n'est pas correct

File: strings/src/main/res/values/untranslated.xml [hunk 3]
```diff
@@ -39,10 +39,15 @@ the specific language governing permissions and limitations under the License. -
     <string name="btn_sondage" translatable="false">Sondage</string>
     <string name="btn_messsage" translatable="false">Forum</string>
     <string name="btn_article" translatable="false">Article</string>
+    <string name="btn_notification" translatable="false">Notification</string>
     <string name="btn_centreHospital" translatable="false">Formation\n Sanitaires</string>
     <string name="btn_cycle" translatable="false">Mon\n Cycle </string>
     <string name="btn_about" translatable="false">About</string>
     <string name="form_selected" translatable="false">(%s)Formulaire sélectionné(s)</string>
+    <string name="get_quest" translatable="false">Télécharger un Questionnaire Vierge</string>
```
Comment (Baom92) [inline strings/src/main/res/values/untranslated.xml]: Pourquoi changer le libellé des boutons ? Utiliser les anciens libellés

**Review by Baom92 — CHANGES_REQUESTED** [review]

File: collect_app/src/main/java/org/odk/collect/android/activities/MainMenuActivity.java [hunk 4]
```diff
@@ -140,7 +140,7 @@ public void onClick(View v) {
 
         // manage forms button. no result expected.
         getFormsButton = findViewById(R.id.get_forms);
-        getFormsButton.setText(getString(R.string.get_forms));
+        getFormsButton.setText(getString(R.string.questionnaire_form));
```
Comment (Baom92) [inline collect_app/src/main/java/org/odk/collect/android/activities/MainMenuActivity.java]: Pourquoi changer le nom ? Garder l'ancien nom

File: collect_app/src/main/res/layout/main_menu.xml [hunk 5]
```diff
@@ -47,40 +47,39 @@
...
-                        style="@style/Widget.Collect.Button.BlocButton"
-                        android:text="@string/review_data"
-                        app:icon="@drawable/ic_edit_note"
-                        tools:text="@string/review_data" />
+                <org.odk.collect.androidshared.ui.multiclicksafe.MultiClickSafeButton
+                    android:id="@+id/get_forms"
+                    style="@style/Widget.Collect.Button.BlocButton"
+                    android:text="@string/questionnaire_form"
+                    app:icon="@drawable/ic_save"
+                    tools:text="@string/questionnaire_form" />
```
Comment (Baom92) [inline collect_app/src/main/res/layout/main_menu.xml]: Garder l'ancien texte

File: collect_app/src/main/res/layout/main_menu.xml [hunk 6]
```diff
@@ -47,40 +47,39 @@
...
+
+                <org.odk.collect.androidshared.ui.multiclicksafe.MultiClickSafeButton
+                    android:id="@+id/enter_data"
+                    style="@style/Widget.Collect.Button.BlocButton"
+                    android:text="@string/fill_quest"
+                    app:icon="@drawable/ic_formulaire"
+                    tools:text="@string/fill_quest" />
```
Comment (Baom92) [inline collect_app/src/main/res/layout/main_menu.xml]: Garder l'ancien texte

File: strings/src/main/res/values/untranslated.xml [hunk 7]
```diff
@@ -39,10 +39,16 @@ the specific language governing permissions and limitations under the License. -
     <string name="btn_sondage" translatable="false">Sondage</string>
     <string name="btn_messsage" translatable="false">Forum</string>
     <string name="btn_article" translatable="false">Article</string>
+    <string name="btn_notification" translatable="false">Notification</string>
     <string name="btn_centreHospital" translatable="false">Formation\n Sanitaires</string>
     <string name="btn_cycle" translatable="false">Mon\n Cycle </string>
     <string name="btn_about" translatable="false">About</string>
     <string name="form_selected" translatable="false">(%s)Formulaire sélectionné(s)</string>
-    <string name="upload_answer" translatable="false">Envoie du Formulaire</string>
+    <string name="questionnaire_form" translatable="false">Télécharger un Questionnaire Vierge</string>
```
Comment (Baom92) [inline strings/src/main/res/values/untranslated.xml]: A supprimer. Garder l'ancien texte pour le bouton

File: strings/src/main/res/values/untranslated.xml [hunk 8]
```diff
@@ -39,10 +39,16 @@ the specific language governing permissions and limitations under the License. -
...
+    <string name="fill_quest" translatable="false">Remplir un Questionnaire</string>
```
Comment (Baom92) [inline strings/src/main/res/values/untranslated.xml]: A supprimer. Garder l'ancien texte pour le bouton

**Review by Baom92 — CHANGES_REQUESTED** [review]

**Review by Baom92 — APPROVED** [review]

**Review by jgnacadja — APPROVED** [review]

**Pull request merged**